backend/database/factories/EngineeringDocumentFactory.php

```
@@ -0,0 +1,43 @@
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\EngineeringDocument>
+ */
+class EngineeringDocumentFactory extends Factory
+{
+    public function definition(): array
+    {
+        $uuid = Str::uuid()->toString();
+
+        return [
+            'entity_type' => 'material',
+            'entity_id' => Material::factory(),
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\Material::factory().


(staticMethod.notFound)